.github/ISSUE_TEMPLATE.md

```diff
@@ -9,11 +9,11 @@
 
 ### Bug reports:
 
-Please replace this line with a brief summary of your issue **AND** if a stack trace / screenshot of the exception you are facing:
+Please replace this line with a brief summary of your issue **AND** a stack trace / screenshot of the exception you are facing.
 
 ### Features:
 
-Tell us briefly, what your feature request is about, which problem it solves, and whether you want to help implementing it.
+Tell us briefly, what your feature request is about, which problem it solves, and whether you want to help implement it.
```

Comment:
```suggestion
Tell us briefly, what your feature request is about, what problem it solves, and whether you want to help implement it.
```
the implement was good :+1: 
just saw that:
https://english.stackexchange.com/questions/3413/which-vs-what-whats-the-difference-and-when-should-you-use-one-or-the-ot